### PR TITLE
ensure BOM creation is quiet when debug=False

### PR DESCRIPTION
**Pull request by luxe**
Some of the noise is coming from `licensetool.py`.  Added a flag to set its log level.
fixes: https://github.com/vmware/rules_oss_audit/issues/6

### CODE REVIEW

File: oss_audit/java/oss_audit.bzl [hunk 1]
```diff
@@ -187,7 +194,7 @@ def _oss_audit_impl(ctx):
                 artifact_id,
                 name,
                 srcjar_url if srcjar_url else "",
-                version,
+                version
```
Comment (danielmachlab) [inline oss_audit/java/oss_audit.bzl:197]: What is the reason for removing this `,`? buildifier is generating warnings from it.

File: oss_audit/tools/licensetool.py [hunk 2]
```diff
@@ -139,8 +139,10 @@ def main():
     )
     parser.add_argument("url", help="Jar file location url")
     parser.add_argument("output", help="Output file")
+    parser.add_argument("--log_level", required=False, default="ERROR",help="Provide logging level. Example --log_level debug', default='warning'")
```
Comment (danielmachlab) [inline oss_audit/tools/licensetool.py:142]:
```suggestion
    parser.add_argument("--log_level", required=False, default="ERROR",help="Provide logging level. Example --log_level debug")
```
Does this look okay? Perhaps there is an extra `default='warning'` in there.

**Review by danielmachlab — CHANGES_REQUESTED** [review]
Hi Trevor, thanks for making these changes. `oss_audit` runs much quieter now, which is nice! Two small comments before we merge this in.

File: oss_audit/java/oss_audit.bzl (hunk 1 above)
Comment (luxe) [reply oss_audit/java/oss_audit.bzl]: accident. fixed.

File: oss_audit/tools/licensetool.py (hunk 2 above)
Comment (luxe) [reply oss_audit/tools/licensetool.py]: that's better.  licensetool.py only uses info/debug.  changed it to WARNING.  python throws if its not all caps.

Comment (danielmachlab) [reply oss_audit/tools/licensetool.py]: Looks good. Thanks

**Review by danielmachlab — APPROVED** [review]

**Pull request merged**